### DISCUSSION TITLE
Are extendedProperties supported when you create an item?

### DISCUSSION THREAD

**Discussion by WilcoHendriksen** [microsoft/Microsoft-Fabric-workload-development-sample#168] [Q&A] [answered]
Are [extendedProperties ](https://learn.microsoft.com/en-us/javascript/api/@ms-fabric/workload-client/createitempayload?view=fabric-extensibility-js-latest#@ms-fabric-workload-client-createitempayload-extendedproperties) supported when you create an item?

And if they are, are they included in itemCRUD.getItem response?

The response of the artifact request:
```json
{
    "error": {
        "code": "ArtifactInputFailure",
        "pbi.error": {
            "code": "ArtifactInputFailure",
            "parameters": {
                "FieldName": "ExtendedProperties",
                "FailureReason": "Extended Property with name asd is invalid."
            },
            "details": [],
            "exceptionCulprit": 1
        }
    }
}
```

**Comment by IdanGoldfarb1989** [comment, marked as answer]
No, it's not supported.
